Source/CombatExtended/Harmony/Harmony_Fire.cs

```diff
@@ -125,6 +125,41 @@ internal static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstruct
                 }
             }
         }
+        internal static void Postfix(Fire __instance)
+        {
+            if (__instance.Spawned && __instance.parent is Pawn pawn)
+            {
+                __instance.fireSize -= pawn.GetStatValue(StatDefOf.Flammability, true) * 150 * 0.00055f;
```

Comment:
Is there a rationale behind the `0.00055f` ? I notice it comes up twice?

Reply:
Hard to say, given that C# embeds constants where they are used.  That constant appears in the disassembly for `Fire.cs` in the RW sources.  The first use here is to undo the stock fire growth, the second is to apply the calculated fire growth mount.